modules/nested-elements/assets/js/editor/nested-tabs/commands/select.js

```diff
@@ -0,0 +1,67 @@
+import CommandHistory from 'elementor-document/commands/base/command-history';
+
+export class Select extends CommandHistory {
```

Comment:
It should be under `nested-repeater`, in order to support extending for other widgets

Reply:
Handled.